app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.user = current_user
+    # authorize @comment
```

Comment:
@maciek1995 jak wrzucam authorize to wyskakuje mi błąd `wrong number of arguments (given 2, expected 0)`, pomysł co może być przyczyną?